website/content/en/docs/building-operators/golang/advanced-topics.md

```diff
@@ -28,7 +28,14 @@ type MyAppStatus struct {
 }
 ```
 
+<!-- todo(camilamacedo) 
+the following link was comment for do no broke the CI
+this impl probably will came from operator-lib
+however, we also need to check if we would not like to just let them use
+https://github.com/kubernetes/kubernetes/pull/92717/files 
```

Comment:
`pkg/status` was also moved to `operator-lib`. Let's just update that link as well.

And I don't think this is the appropriate place for a comment about the choice of implementations to use.

I think a better choice would be to open an issue in `operator-lib` asking whether it makes sense to deprecate `status` and point users to `k8s.io/apimachinery`'s implementation.